src/main/java/br/edu/unicesumar/folia/controller/boleto/BoletoDTO.java

```diff
@@ -0,0 +1,4 @@
+package br.edu.unicesumar.folia.controller.boleto;
+
+public class BoletoDTO {
```

Comment:
Se não for usar você remove !

Reply:
Boleto tambem feito pelo fernandão

Reply:
Remove nao tem codigo !

 se não tem codigo não há o por que de existir.